src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -129,6 +135,9 @@ public abstract class PeerConnector : IPeerConnector
             this.NodeSettings = nodeSettings;
             this.peerAddressManager = peerAddressManager;
             this.Requirements = new NetworkPeerRequirement { MinVersion = this.NodeSettings.ProtocolVersion };
+
+            this.DefaultConnectionAttemptInterval = TimeSpans.Second;
```

Comment:
@noescape00 how about we rename this to `defaultConnectionInterval`, make it a constant and create a second constant called `burstConnectionInterval`? Then use them where appropriate?

Thoughts?

Reply:
Sounds good to me. 
Fixed. 

ps
made them readonly because it's not possible to assign constant to non-compile time value (TimeSpan.Zero)